frontend/src/app/modules/edit-claim/edit-claim-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
This will be housing specific route for edit claim after everything is in its own module - cleans up the app-module-routing module.